src/containers/onboarding/AddAccountPage.tsx

```diff
@@ -56,25 +53,13 @@ const stylesDecorator = withStyles(styles, {
 @inject('langStore')
 @inject('onboardingStore')
 @inject('walletStore')
-@inject('ledgerStore')
 @observer
 class AddAccountPage extends React.Component<Props> {
-  private ledger: LedgerChannel;
 
   get injected(): PropsInjected {
     return this.props as PropsInjected;
   }
 
-  componentWillMount() {
-    const { ledgerStore } = this.injected;
-    this.ledger = ledgerStore.openChannel();
-  }
-
-  componentWillUnmount() {
-    // pass async
-    this.ledger.close();
-  }
```

Comment:
Fyi, opening the Ledger here was an UX optimization. That way when the user navigated into adding a Ledger account, the "connect ledger" wouldn't be shown for n-seconds it took to properly ping the device.